src/macaron/slsa_analyzer/registry.py

```diff
@@ -498,25 +677,42 @@ def prepare(self) -> bool:
         """
         self._init_runners()
 
+        # Only support 1 runner at the moment.
+        if not self.runners or len(self.runners) != 1:
+            logger.critical("Invalid number of runners.")
+            return False
+
+        if not self._all_checks_mapping:
+            logger.error("Cannot run because there is no check registered.")
+            return False
+
         try:
             if not self._is_graph_ready:
                 self._graph.prepare()
                 self._is_graph_ready = True
-
-            # Only support 1 runner at the moment.
-            if not self.runners or len(self.runners) != 1:
-                logger.critical("Invalid number of runners.")
-                return False
-
-            if not self._all_checks_mapping:
-                logger.error("Cannot run because there is no check registered.")
-                return False
-
-            return True
         except CycleError as error:
             logger.error("Found circular dependencies in registered checks: %s", str(error))
             return False
 
+        ex_pats = defaults.get_list(section="analysis.checks", item="exclude", fallback=[])
+        in_pats = defaults.get_list(section="analysis.checks", item="include", fallback=["*"])
+        try:
+            checks_to_run = registry.get_final_checks(ex_pats, in_pats)
+        except CheckRegistryError as error:
+            logger.error(error)
+            return False
+
+        if len(checks_to_run) == 0:
+            logger.info("There is no check to run according to the exclude/include configuration.")
```

Comment:
```suggestion
            logger.info("There are no checks to run according to the exclude/include configuration.")
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/254/commits/63b2883b703ee2b96584f00cb34f560f49a87db9